coordinator/src/orderbook/trading.rs

```diff
@@ -224,29 +217,22 @@ async fn process_new_order(
                 },
             };
 
-            let msg = OrderbookMessage::TraderMessage { trader_id, message };
+            let notification = match &order.order_reason {
+                OrderReason::Expired => Some(NotificationKind::PositionExpired),
+                OrderReason::Manual => None,
```

Comment:
What's a notification of `None`?

Reply:
Maybe too implicit? But the idea is if you don't set a notification Kind it means there should be not push notification sent if the user is not online.

The idea is to have optional notification kinds that will only get sent if the user is offline.

Reply:
Right, that was the other code I've refactored. Thank you for the clarification. Can we add this to the message so that it's clear what behavior to expect?